### PR TITLE
[formrecognizer] Fix documentation

### LINKED ISSUE #23973: [formrecognizer] Missing bracket in docs
Add missing bracket in docs:
https://github.com/Azure/azure-sdk-for-python/blob/dcfad094300df3051412720e5a3b16c4c5454aff/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py#L770

https://github.com/Azure/azure-sdk-for-python/blob/dcfad094300df3051412720e5a3b16c4c5454aff/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py#L854

### PR DESCRIPTION
**Pull request by catalinaperalta**
Fixes #23973 
Adds missing brackets in docs.

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def begin_recognize_receipts(self, receipt, **kwargs):
             and/or ranges of pages you want to get in the result. For a range of pages, use a hyphen, like
             `pages=["1-3", "5-6"]`. Separate each page number or range with a comma.
         :return: An instance of an LROPoller. Call `result()` on the poller
-            object to return a list[:class:`~azure.ai.formrecognizer.RecognizedForm`].
+            object to return a list[:class:`~azure.ai.formrecognizer.RecognizedForm`]].
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py:114]: doesn't look like we need the extra bracket here or am I missing something?

edit: it seems like only the rtype needs correction?

Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py:114]: Oops! A find error. Fixing

Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py]: Done

**Review by kristapratico — APPROVED** [review]
good catch!

**Pull request merged**